services/src/test/scala/cromwell/pipeline/service/ProjectServiceTest.scala

```diff
@@ -35,6 +42,23 @@ class ProjectServiceTest extends AsyncWordSpec with Matchers with MockitoSugar w
       }
     }
 
+    "updateProjectName" should {
+      "return ok of a new project name" taggedAs Service in {
```

Comment:
```suggestion
      "return projectId if update was successful" taggedAs Service in {
```
Sorry, I didn't notice it last time. I can't understand what this test says :(

Reply:
My fault. Because I am a copy-paste master :(